crates/walrus-sdk/src/config/communication_config.rs

```diff
@@ -62,6 +62,9 @@ pub struct ClientCommunicationConfig {
     #[serde(rename = "sui_client_request_timeout_millis")]
     #[serde_as(as = "Option<DurationMilliSeconds>")]
     pub sui_client_request_timeout: Option<Duration>,
+    /// The maximum time to wait for a blob to be certified.
+    #[serde(rename = "read_blob_certification_wait_time_secs")]
+    pub read_blob_certification_wait_time: u64,
```

Comment:
Suggestion: Use a `Duration` here similar to above with a `#[serde_as(as = "DurationSeconds")]` annotation.